### DISCUSSION TITLE
Feature request: auto download liked

### DISCUSSION THREAD

**Discussion by BrowneGit** [UnicornsOnLSD/finamp#466] [General]
Not sure if this is the right spot to request a feature.
Would love to have a setting to have liked tracks/albums/artists saved to local storage (download) automatically.
ie, if I click the heart on a song and have the setting turned on in the menu it will automatically download that song.
Love the app btw, thank you

**Comment by UnicornsOnLSD** [comment]
This is being tracked in #146 :)
